### PR TITLE
feat(dsg): update packages in generate code

### PR DESCRIPTION
**Pull request by tupe12334**
Close: #5663 

## PR Details

This pull request changes the `package.templat.json` files to `package.json` files in order to apply dependabot on them.

## PR Checklist

- [x] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

File: packages/data-service-generator/src/admin/package-json/package.json [hunk 1]
```diff
@@ -29,10 +29,17 @@
...
+    ]
   },
   "browserslist": {
-    "production": [">0.2%", "not dead", "not op_mini all"],
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
     "development": [
```
Comment (overbit) [inline packages/data-service-generator/src/admin/package-json/package.json:43]: Do we need to run the `npx nx format`?

File: packages/data-service-generator/src/admin/package-json/create-package-json.ts [hunk 2]
```diff
@@ -12,13 +12,12 @@ import { paramCase } from "param-case";
 import { promises as fs } from "fs";
 
 const PACKAGE_JSON_ENCODING = "utf-8";
-const PACKAGE_JSON_TEMPLATE = "package.template.json";
 const PACKAGE_JSON_FILE_NAME = "package.json";
 
-const filePath = resolve(__dirname, PACKAGE_JSON_TEMPLATE);
+const filePath = resolve(__dirname, PACKAGE_JSON_FILE_NAME);
```
Comment (mulygottlieb) [inline packages/data-service-generator/src/admin/package-json/create-package-json.ts:17]: I am of course not familiar enough with this code, but wouldn't reading a different file (not the .template one) change the functional meaning of this code?

File: packages/data-service-generator/src/server/package-json/create-package-json.ts [hunk 3]
```diff
@@ -12,13 +12,12 @@ import { paramCase } from "param-case";
...
 const PACKAGE_JSON_ENCODING = "utf-8";
-const PACKAGE_JSON_TEMPLATE = "package.template.json";
 const PACKAGE_JSON_FILE_NAME = "package.json";
 
-const filePath = resolve(__dirname, PACKAGE_JSON_TEMPLATE);
+const filePath = resolve(__dirname, PACKAGE_JSON_FILE_NAME);
 
 export async function createServerPackageJson(): Promise<Module[]> {
-  const fileContent = await fs.readFile(filePath, "utf-8");
+  const fileContent = await fs.readFile(filePath, PACKAGE_JSON_ENCODING);
```
Comment (mulygottlieb) [inline packages/data-service-generator/src/server/package-json/create-package-json.ts:20]: Same question here - wouldn't reading a different file change the functionality?

File: packages/data-service-generator/src/admin/package-json/create-package-json.ts (hunk 2 above)
Comment (tupe12334) [reply packages/data-service-generator/src/admin/package-json/create-package-json.ts:17]: We change the `package.template.json` to called `package.json` in order to let the dependabot recognize this file.

**Review by mulygottlieb — APPROVED** [review]

**Pull request merged**